src/main/java/com/example/icecreamup/IcecreamupController.java

```diff
@@ -23,7 +23,7 @@
 public class IcecreamupController {
 	
 	@Autowired
-	private ITopOrderService topOrderService;
+	private ITopOrderService topOrderServiceStub;
```

Comment:
The name of this variable should reflect the class it is going to be Autowired to